src/AggregateFunctions/registerAggregateFunctions.cpp

```diff
@@ -82,6 +82,7 @@ void registerAggregateFunctionIntervalLengthSum(AggregateFunctionFactory &);
 void registerAggregateFunctionAnalysisOfVariance(AggregateFunctionFactory &);
 void registerAggregateFunctionFlameGraph(AggregateFunctionFactory &);
 void registerAggregateFunctionKolmogorovSmirnovTest(AggregateFunctionFactory & factory);
+void registerAggregateFunctionGroupArraySorted(AggregateFunctionFactory & factory);
```

Comment:
a) please move this up to the other groupArray related functions b) if possible, please sort l. 12ff. Having everyone add new stuff to the end is a merge conflict generator.